.papr.yml

```diff
@@ -50,8 +69,6 @@ cluster:
 context: centos/7/atomic
 
 tests:
-  # skipping kernel_avc_denied due to https://bugzilla.redhat.com/show_bug.cgi?id=1536991
-  #  - should be fixed in CentOS 7.5
   # skipping 'default_t' checks until atomic 1.22 lands with (projectatomic/atomic#1185)
-  - ./.test_director --skip-tags kernel_avc_denied,default_file_label
```

Comment:
Minor: should this be a separate commit? I don't see anything about it in the commit msg.

Reply:
Sure, that would be the proper thing to do